packages/snaps-sdk/tsconfig.json

```diff
@@ -3,12 +3,13 @@
   "compilerOptions": {
     "baseUrl": "./",
     "jsx": "react-jsx",
-    "jsxImportSource": "@metamask/snaps-sdk"
+    "jsxImportSource": "@metamask/snaps-sdk",
+    "module": "ES2020",
+    "moduleResolution": "Node10"
```

Comment:
`snaps-sdk` has a `.ts` file which declares `*.svg`, `*.png`, etc., but this doesn't work in `Node16` or `NodeNext` module resolution. I've kept `snaps-sdk` and the image example at `Node10` for now, to not introduce any breaking changes.